dotCMS/src/main/java/com/dotmarketing/db/listeners/CommitListener.java

```diff
@@ -0,0 +1,28 @@
+package com.dotmarketing.db.listeners;
+
+public abstract class CommitListener implements DotListener {
+
+    @Override
+    public final boolean equals(Object obj) {
+        
+        
+        if(obj == null ||!( obj instanceof CommitListener)) {
+            return false;
+        }
+        CommitListener other = (CommitListener)obj;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'other' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)